### PR TITLE
Reorder layer objects when their object menu is closed

### PR DESCRIPTION
**Pull request by Hume2**

### CODE REVIEW

File: src/editor/layers_gui.cpp [hunk 1]
```diff
@@ -255,4 +260,9 @@ EditorLayersGui::get_layer_pos(const Vector& coords){
   return (coords.x - Xpos - sector_text_width) / 35;
 }
 
+bool
+EditorLayersGui::less_z_pos(const std::unique_ptr<LayerIcon>& lhs, const std::unique_ptr<LayerIcon>& rhs) {
```
Comment (tobbi) [inline src/editor/layers_gui.cpp]: This can be marked const afaict.

Comment (Hume2) [inline src/editor/layers_gui.cpp]: @tobbi Static functions can't be marked const.

Comment (tobbi) [inline src/editor/layers_gui.cpp]: Right, didn't see it was static. It's alright then.

**Comment by tobbi** [comment]
lgtm

**Comment by tobbi** [comment]
Actually, now that I think about it:
Can we reverse the order of the layers somehow. I think the minus layers should be left and the plus layers should be right.

**Pull request merged**